gitgot.py

```diff
@@ -134,6 +134,12 @@ def should_parse(repo, state, is_gist=False):
             "API Error: File no longer exists on github.com" +
             bcolors.ENDC)
         return False
+    except github.GithubException:
```

Comment:
This is the base exception class used by PyGithub. So, this could mean a lot of things. Ideally we'd submit a PR to PyGithub first to enable a specific exception for this type of issue. But it might be prudent to just print out the exception and continue. Thoughts?

Reply:
That make sense. I updated the error message to print the exception message.